lib/assessment.rb

```diff
@@ -43,6 +59,13 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    #I don't really understand the ARGV[0], just took from read-file.rb demo yesterday, and I don't know when to use it/if I need it here.
+    File.open(ARGV[0]) do |filename|
```

Comment:
When you use File.open, what should you pass in as an argument?  And when it does its iteration, what are the specific things that it's iterating through?